.github/workflows/muon.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: nvme-cli muon
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build-minimal:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: build muon
+        run: |
+          mkdir build-tools
+          cd build-tools
+          git clone --depth 1 https://git.sr.ht/~lattis/muon
+          cd muon
+
+          export CC=gcc
+          export ninja=build/samu
+
+          ./tools/bootstrap_ninja.sh build
+          ./bootstrap.sh build
+
+          build/muon setup -Dlibcurl=disabled -Dlibarchive=disabled -Ddocs=disabled -Dsamurai=disabled build
+          build/samu -C build
+          build/muon -C build test
+      - name: fetch libnvme
+        run: |
+          cd subprojects
+          git clone https://github.com/linux-nvme/libnvme.git
+          # libnvme_ref=$(sed -n "s/revision = \([0-9a-z]\+\)/\1/p" libnvme.wrap)
+          # git -C libnvme checkout $libnvme_ref
```

Comment:
Need to figure out why this part doesn't work. The version check for libnvme fails in the main meson.build file.

Reply:
FTR, the problem was that muon somehow had troubles with a ref which also got a tag pointing to. It works fine between the releases. Ah, something to figure out later :)